cluster-autoscaler/apis/provisioningrequest/autoscaling.x-k8s.io/v1beta1/types.go

```diff
@@ -200,4 +200,8 @@ const (
 	// ProvisioningClassBestEffortAtomicScaleUp denotes that CA try to provision the capacity
 	// in an atomic manner.
 	ProvisioningClassBestEffortAtomicScaleUp string = "best-effort-atomic-scale-up.autoscaling.x-k8s.io"
+	// ProvisioningRequestPodAnnotationKey is a key used to annotate pods consuming provisioning request.
```

Comment:
Please bump the API version in CA's go.mod so that the dependency on the new fields is expressed properly.

Reply:
Bumped using:
```
$ go get k8s.io/autoscaler/cluster-autoscaler/apis
```